src/main.rs

```diff
@@ -254,7 +254,7 @@ fn main() {
         let path_mapping_file = path_mapping_file.to_owned();
         let path_mapping = Arc::clone(&path_mapping);
 
-        thread::spawn(move || {
+        thread::Builder::new().name(format!("Producer")).spawn(move || {
```

Comment:
No need for format! here, since this is not a formatting string (it doesn't contain any `{}`)

Reply:
done